app/_src/mesh/features/openshift-quickstart.md

```diff
@@ -0,0 +1,197 @@
+* How to deploy a sample application, `kuma-demo`, on the mesh and validate that the application is working
+* How to use the sample application to test the features of {{site.mesh_product_name}}
+
+## Prerequisites
+* [ROSA CLI installed](https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-installing-rosa.html) or another OpenShift 4.12 cluster with the ability to create LoadBalancer type Kubernetes Services
+* [An AWS account with ROSA permissions enabled](https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-config-aws-account.html)
+* [kubectl CLI installed](https://kubernetes.io/docs/tasks/tools/)
+* [OpenShift Container Platform CLI installed](https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html)
+* [Helm 3 installed](https://helm.sh/docs/intro/install/)
+* Clone the {{site.mesh_product_name}} OpenShift quickstart repository:
+    ```bash
+    git clone https://github.com/Kong/kong-mesh-quickstart-openshift.git
+    ```
+* Clone the `kuma-demo` app:
+    ```bash
+    git clone https://github.com/kumahq/kuma-counter-demo.git
+    ```
+
+## Install ROSA
+
+In this section, you will install a ROSA cluster called `kong-mesh-2` in the `us-west-2` region. Then, you'll create an admin user to get quick access to the cluster. 
+
+1. Create system variables for the cluster name and region:
+    ```bash
+    CLUSTER_NAME=kong-mesh-2
+    REGION=us-west-2
+    ```
+
+1. Create a small ROSA cluster:
+    ```bash
+    rosa create cluster --cluster-name=$CLUSTER_NAME --region=$REGION --multi-az=false --version 4.12.13
+    ```
+
+1. When the cluster install is complete, create a cluster-admin user:
+    ```bash
+    rosa create admin --cluster $CLUSTER_NAME
+    ```
+    Creating an admin user gives you quick access to the cluster for this quickstart. We recommend using a formal identity provider that you grant admin privileges to in a production environment.
+
+1. Validate you can log in to the cluster via the credentials provided by the ROSA CLI `stout`. Once you successfully login, you can proceed to the next section.
+
+## Install {{site.mesh_product_name}} in standalone mode
+
+In this section, you'll install {{site.mesh_product_name}} in standalone mode. Standalone mode is used in this quickstart for simplicity. 
+
+1. Create the `kong-mesh-system` namespace:
+    ```bash
+    kubectl create namespace kong-mesh-system
+    ```
+
+1. Create the image pull secret:
+    ```bash
+    kubectl create secret docker-registry rh-registry-secret -n kong-mesh-system \
+        --docker-server=registry.connect.redhat.com \
+        --docker-username=<username> \
+        --docker-password=<password> \
+        --docker-email=<email>
+    ```
+
+    Doing this allows you to authenticate so you can pull down the images from Red Hat's repository.
```

Comment:
consider rephrase: "Doing this authenticates you to Red Hat's image registry, which allows you to pull the certified images"